modules/account-lib/src/coin/celo/transactionBuilder.ts

```diff
@@ -52,6 +52,9 @@ export class TransactionBuilder extends Eth.TransactionBuilder {
       const txData = JSON.parse(rawTransaction);
       tx = new Transaction(this._coinConfig, this._common, txData);
     }
+    // TODO: STLX-7111 Analyze Celo breaks behavior of EthLike enable v feature
+    // Celo has enabled v by default as chainId. This prevent override behavior.
+    super.enableV(false);
```

Comment:
so yeah, again, let's call this `enableVFieldChecking` or something to that effect

Reply:
it is refactored into enableFinalVField